### PR TITLE
Old Makefile functionaliy

### PR DESCRIPTION
**Pull request by c71n93**
- Returned `init` target to `Makefile`
- Edited `all` target

### CODE REVIEW

**Review by levBagryansky — DISMISSED** [review]
@c71n93 Thanks

File: Makefile [hunk 1]
```diff
@@ -10,8 +10,12 @@ RISCV_SYSROOT ?= $(PWD)/../sysroot
 JOBS ?= 8
 
 .PHONY: all
-all: test test-e2e
+all: init format test test-e2e
 
+.PHONY: init
+init:
+	git submodule init
+	git submodule update
```
Comment (mmamayka) [inline Makefile:18]: `git submodule update --init --recursive` is a proper command

**Review by mmamayka — CHANGES_REQUESTED** [review]

**Comment by c71n93** [comment]
@mmamayka @levBagryansky check

**Review by levBagryansky — APPROVED** [review]

**Review by mmamayka — APPROVED** [review]
LGBT

**Pull request merged**